R/pkg/R/SQLContext.R

```diff
@@ -186,6 +186,8 @@ getDefaultSqlSource <- function() {
 #'
 #' @param data an RDD or list or data.frame.
 #' @param schema a list of column names or named list (StructType), optional.
+#' @param samplingRatio Currently not used.
+#' @param numPartitions the number of partitions of the SparkDataFrame.
```

Comment:
Related to the comment in the test case it might be good to say what the default is here ?